tensorboard/plugins/projector/tf_projector_plugin/index.js

```diff
@@ -22,6 +22,9 @@ iframe {
   height: 100%;
   margin: 0;
   width: 100%;
+}
+body {
+  line-height: 0;
```

Comment:
I think this should be avoided if possible and is something that can be replaced with `display: block`.